css/style.css

```diff
@@ -1,8 +1,122 @@
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- */
+
+ * {
+    box-sizing: border-box;
+  }
+  
+  /* Add a gray background color with some padding */
+  body {
+    font-family: Arial;
+    padding: 20px;
+    background: #eaebee;
+  }
+  
+  /* Header/Blog Title */
+  .header {
+    padding: 30px;
+    font-size: 60px;
+    text-align: center;
+    background: white;
+    color: rgb(112, 80, 80);
+    text-shadow: 2px 2px 8px #272424
+  }
+  .topnav {
+    overflow: hidden;
+    background-color: rgb(92, 81, 81);
+    
```

Comment:
No need for an extra empty line there. This is a very small thing, but in general we try to follow a consistent style when writing code. There probably should not be an empty line before closing the }.